server/constants/validation-constants.ts

```diff
@@ -74,12 +74,14 @@ export const participantStatuses = [
   'rosComplete',
 ];
 
-export const postHireStatuses = {
-  orientationCompleted: 'orientation_completed',
-  postSecondaryEducationUnderway: 'post_secondary_education_underway',
-  postSecondaryEducationCompleted: 'post_secondary_education_completed',
-  cohortUnsuccessful: 'cohort_unsuccessful',
-};
+// Not sure why eslint thinks this is a shadow - it isn't.
+// eslint-disable-next-line no-shadow
+export enum postHireStatuses {
```

Comment:
Can you explain why const -> enums makes sense for these changes?

Reply:
Two reasons:
1. It's more idiomatic. The entire point of an enum is to store a set of named constants, and that's exactly what we're doing here.
2. Stronger type checking. When using enums, we can say "this MUST be one of these values", and TS will do its best to enforce that. Each member is also treated as a unique type of sorts, which can be handy. The main benefit here is enforcing strict values, though - code that takes an enum mandates that invalid values doesn't reach it.

Reply:
Good reasons :)

Reply:
Love this!